examples/gno.land/p/demo/grc/grc721/util.gno

```diff
@@ -0,0 +1,18 @@
+package grc721
+
+import (
+	"std"
+)
+
+var zeroAddress = std.Address("")
+
+func isValidAddress(addr std.Address) error {
+	if addr.String() == "" {
+		return ErrInvalidAddress
+	}
+	return nil
+}
```

Comment:
Don't bach32 addresses need to be of a certain length as well?

Reply:
Additionally, for `isValid...` methods, we should strive to return boolean values, not specific errors, because the usage will almost always be within a conditional statement

Reply:
This imitates [grc/grc20/util.gno](https://github.com/harry-hov/gno/blob/hariom/grc721/examples/gno.land/p/demo/grc/grc20/util.gno). 

I didn't focused on `isValidAddress()` coz I think it should be the part of `std` library. And should be handled in separate PR.